src/core/RecyclerListView.tsx

```diff
@@ -104,7 +104,7 @@ export interface RecyclerListViewProps {
     optimizeForInsertDeleteAnimations?: boolean;
     style?: object | number;
     debugHandlers?: DebugHandlers;
-
+    contentContainer?: (props?: object, children?: React.ReactNode) => JSX.Element | null;
```

Comment:
return react.reactnode

Reply:
change the name to renderContentContainer

Reply:
Done